hr_employee_id/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Employee Identification Numbers
```

Comment:
@pedrobaeza thank you for taking the time to look at this; I don't understand what you mean by this; looked through other READMEs in OCA/hr and they look similar...

P.S. THis is my first OCA contrib so forgive me I am unaware of these seemly apparent conventions